### DISCUSSION TITLE
Current State of the App | Looking for Contributors

### DISCUSSION THREAD

**Discussion by astubenbord** [astubenbord/paperless-mobile#456] [Announcements]
As many of you may have already noticed - either by the lack of updates or some comments I posted - I am _currently_ not in the position to actively work on this app anymore since other private interests, hobbies and my full time job have grown to consume way more time than they did about a year ago. Additionally, when I started developing this app, paperless-ngx was far less popular than it is now: With an increasing number of developers and increasingly more frequent and larger updates, I, as a sole developer, am not able to keep up with the (sometimes breaking) changes introduced by the paperless-ngx team, let alone the daily growing number of reported issues in this repository. I'd go as far as saying that even by just implementing new API features and not fixing existent bugs, I'd not be able to keep up with development.

As much as I would like to keep working on this, out of the aforementioned reasons, in the current state it is simply not possible, at least not in a way it was a year ago. 

That said, since this project has gained quite some popularity amongst paperless-ngx users and has grown way beyond what I originally imagined, I really don't want development to be ceased. I am sometimes still working on it, but development has just become much slower and more sporadic. 

Therefore, I am looking for contributors for this project. So, if you are as enthusiastic about paperless, flutter or (app) development in general, please let me know! I'd be happy for everyone who'd like to contribute, be it in the form of code itself or just helping me manage issues or do troubleshooting (I know some of you are already actively commenting and helping others out in issues, I appreciate y'all :)) 

Best
Anton

**Comment by Tronde** [comment]
Hi @astubenbord,

Thank you for the development and maintenance of this app.

I'm not a developer myself and cannot contribute to this app but maybe you can share some information for folks interested in helping with this project.

- Do you accept contributions in the form of pull requests only or do you seek maintainers who will get permission to merge those requests?
- What's the best way to reach out to you with further questions? Should they just comment here with their questions? Or do you prefer email, chat, etc.?

Best regards,
Joerg

**Reply by astubenbord** [reply]
Hi, sorry for the late reply and thanks for your interest in keeping this project going.

> Do you accept contributions in the form of pull requests only or do you seek maintainers who will get permission to merge those requests?

If anyone is interested in contributing or becoming a maintainer, feel free to contact me (this thread might be a good place to start) and we can talk about how to proceed. I also plan on spending more time on development again. This project is behind both technically (old flutter version) and API-wise (supported api v3 vs. newest v9). Also there's a large pool of issues which need to be prioritized and fixed. There's also some issues with the app's architecture, which was initially built to support all API versions simultaneously, which I think is simply not feasible anymore. I think going with a similar strategy "Immich" follows (i.e. an app version only corresponds to a specific API version) might be the way to go forward, but that's of course open for discussion.

>What's the best way to reach out to you with further questions? Should they just comment here with their questions? Or do you prefer email, chat, etc.?

If you're interested in contributing in any way, just answer this thread and we'll see what's the best way to stay in contact.
